### PR TITLE
InlineFlowTest: Allow for custom `getArgString`

### PR DESCRIPTION
**Pull request by hvitved**
When not using `DefaultFlowTest`, it is useful to be able to implement a custom `getArtString` predicate.

### CODE REVIEW

**Review by jketema — APPROVED** [review]
Do we need to check that this doesn't break any tests for other languages?

Otherwise, LGTM.

**Comment by hvitved** [comment]
> Do we need to check that this doesn't break any tests for other languages?

I have done that now, pushing a dummy "trigger CI" commit, and it worked fine. I'll revert the trigger commit again.

File: shared/dataflow/codeql/dataflow/test/InlineFlowTest.qll [hunk 1]
```diff
@@ -105,12 +112,26 @@ module InlineFlowTestMake<
...
   module DefaultFlowTest = FlowTest<DefaultFlowConfig, DefaultFlowConfig>;
 
+  module ValueFlowTestArgString<DataFlow::ConfigSig ValueFlowConfig, getArgStringSig/2 getArgString>
+  {
+    import FlowTestArgString<ValueFlowConfig, NoFlowConfig, getArgString/2>
+  }
+
   module ValueFlowTest<DataFlow::ConfigSig ValueFlowConfig> {
     import FlowTest<ValueFlowConfig, NoFlowConfig>
   }
```
Comment (yoff) [inline shared/dataflow/codeql/dataflow/test/InlineFlowTest.qll:128]:
```suggestion
  module ValueFlowTest<DataFlow::ConfigSig ValueFlowConfig> {
    import ValueFlowTestArgString<ValueFlowConfig, Impl::getArgString/2>
  }
```
To make the connection to ValueFlowTestArgString clear.

File: shared/dataflow/codeql/dataflow/test/InlineFlowTest.qll [hunk 2]
```diff
@@ -105,12 +112,26 @@ module InlineFlowTestMake<
...
 
+  module TaintFlowTestArgString<DataFlow::ConfigSig TaintFlowConfig, getArgStringSig/2 getArgString>
+  {
+    import FlowTestArgString<NoFlowConfig, TaintFlowConfig, getArgString/2>
+  }
+
   module TaintFlowTest<DataFlow::ConfigSig TaintFlowConfig> {
     import FlowTest<NoFlowConfig, TaintFlowConfig>
   }
```
Comment (yoff) [inline shared/dataflow/codeql/dataflow/test/InlineFlowTest.qll:137]:
```suggestion
  module TaintFlowTest<DataFlow::ConfigSig TaintFlowConfig> {
    import TaintFlowTestArgString<NoFlowConfig, Impl::getArgString/2>
  }
```
To make the connection to TaintFlowTestArgString clear.

**Review by MathiasVP — DISMISSED** [review]

**Review by yoff — DISMISSED** [review]
Python 👍 

I made two suggestions. They make the diff a little larger, but I think they express the intent better. If at some point `ValueFlowTest` should be updated to no longer use `NoFlowConfig`, it can be done in one central place. This is an imaginary example, though, and I do not know how likely such changes are, so I will approve even without it.

**Review by jketema — APPROVED** [review]

**Pull request merged**